### PR TITLE
Squid 6.8 => 7.6

### PR DESCRIPTION
**Pull request by uberhacker**
Tested & Working properly:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l`
### Run the following to get this pull request's changes locally for testing.
```bash
CREW_REPO=https://github.com/chromebrew/chromebrew.git CREW_BRANCH=update-squid crew update \
&& yes | crew upgrade
```

### CODE REVIEW

File: packages/squid.rb [hunk 1]
```diff
@@ -3,34 +3,59 @@
...
+  depends_on 'libxcrypt' => :library
+  depends_on 'libxml2' => :library
+  depends_on 'linux_pam' => :executable
+  depends_on 'linux_pam' => :library
+  depends_on 'nettle' => :executable
+  depends_on 'nettle' => :library
+  depends_on 'openldap' => :executable
+  depends_on 'openldap' => :library
+  depends_on 'openssl' => :executable
+  depends_on 'openssl' => :library
```
Comment (Zopolis4) [inline packages/squid.rb]: duplicate dependencies?

Comment (uberhacker) [reply packages/squid.rb]: Good call.  Fixed.  I thought you were going to mention the missing binaries.  For some reason, they decided to remove them in this release.  No compile option to add them back either.

Comment (uberhacker) [reply packages/squid.rb]: Should we add a test for duplicates?

Comment (Zopolis4) [reply packages/squid.rb]: It's something that will eventually automatically be handled by getrealdeps.rb, but the design is a bit in flux. At the moment, they are more of a visual annoyance in package files than anything serious.

**Review by Zopolis4 — APPROVED** [review]
